README.md

```diff
@@ -87,7 +103,7 @@ To test the build, run `cargo test`.
   cargo test --target x86_64-pc-windows-gnu
   ```
 
-  For `x86`, 
+  For 32-bit R (<=4.1.x), 
```

Comment:
Here and until the end of the file -- I think when `<= 4.1.x` or similar version requirement is written with spaces around operator, it is more readable, so consider adding spaces in all such cases.